components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx

```diff
@@ -98,6 +99,13 @@ export const PrebuildDetailPage: FC = () => {
         };
     }, [prebuildId, disposeStreamingLogs]);
 
+    useEffect(() => {
+        const anyLogAvailable = prebuild?.status?.taskLogs.some((t) => t.logUrl);
+        if (!anyLogAvailable) {
+            setLogNotFound(true);
```

Comment:
Maybe we need to change the not found component's message, 

i.e. `No found` ... instead of `inaccessible`

Reply:
I think in this case it makes sense to log "Inaccessible", because there are no logs to be accessed. I think it's better than "Not found", because it's more generic and we can also apply the message when we get an error from the logs loading.

Reply:
How about no logs found (or it's the same with not found 😅)? `inaccessible` makes me feel like we're not able to access it instead of there's no logs for it

Reply:
let's don't worry about this, customers will tell us if they feel something wrong

Reply:
> How about no logs found (or it's the same with not found 😅)? inaccessible makes me feel like we're not able to access it instead of there's no logs for it

Let's indeed not tackle this now, we can always introduce a more fine-grained error message, but I wanted to work around the added complexity that brings.